src/sagemaker/model.py

```diff
@@ -572,7 +581,8 @@ def compile(
             framework (str): The framework that is used to train the original
                 model. Allowed values: 'mxnet', 'tensorflow', 'keras', 'pytorch',
                 'onnx', 'xgboost'
-            framework_version (str):
+            framework_version (str):The version of framework, for example:
+                '1.5' for PyTorch
```

Comment:
space after colon. also, use e.g.?
```suggestion
            framework_version (str): The version of framework (e.g. '1.5' for PyTorch)
```

Reply:
Thanks!! Addressed.